packages/utils/src/carequality/e2e-validation.ts

```diff
@@ -0,0 +1,210 @@
+import * as dotenv from "dotenv";
+dotenv.config();
+// keep that ^ on top
+import { getEnvVarOrFail } from "@metriport/core/util/env-var";
+import { MetriportMedicalApi, USState, PatientCreate } from "@metriport/api-sdk";
+import { Sequelize, QueryTypes } from "sequelize";
+import { sleep } from "@metriport/shared";
+import dayjs from "dayjs";
+import duration from "dayjs/plugin/duration";
+import {
+  pollOutboundDocQueryResults,
+  pollOutboundDocRetrievalResults,
+} from "@metriport/core/external/carequality/ihe-gateway/poll-outbound-results";
+import axios from "axios";
+
+dayjs.extend(duration);
+
+const apiUrl = getEnvVarOrFail("API_URL");
+const apiKey = getEnvVarOrFail("API_KEY");
+const sqlDBCreds = getEnvVarOrFail("DB_CREDS");
+const cxId = getEnvVarOrFail("CX_ID");
+const cqPatientDataTableName = "cq_patient_data";
+const patientDiscoveryResultTableName = "patient_discovery_result";
+
+const dbCreds = JSON.parse(sqlDBCreds);
+
+const DOC_STATUS_SLEEP = dayjs.duration({ seconds: 30 });
+const PATIENT_SLEEP = dayjs.duration({ seconds: 30 });
+
+const facilityId: string = ""; // eslint-disable-line @typescript-eslint/no-inferrable-types
+
+const metriportAPI = new MetriportMedicalApi(apiKey, {
+  baseAddress: apiUrl,
+});
+
+export const internalApi = axios.create({
+  baseURL: apiUrl,
+  headers: { "x-api-key": apiKey, "Content-Type": "application/json" },
+});
+
+const sequelize = new Sequelize(dbCreds.dbname, dbCreds.username, dbCreds.password, {
+  host: dbCreds.host,
+  port: dbCreds.port,
+  dialect: dbCreds.engine,
+});
+
+const testPatient: PatientCreate = {
+  firstName: "Don",
+  lastName: "Bassett",
+  dob: "1958-04-06",
+  genderAtBirth: "M",
```

Comment:
Why this demo in specific? Aren't there others we should be using?

Reply:
Updated to Innovar one